server/stt/io/feeding_services/stt_content_api.py

```diff
@@ -0,0 +1,282 @@
+from __future__ import annotations
+
+
+import logging
+from datetime import datetime
+from typing import Any, Dict, Iterable, List, Optional, Tuple
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util import Retry
+
+from superdesk.errors import IngestApiError, ParserError
+from superdesk.io.registry import register_feeding_service
+from superdesk.io.feeding_services.http_base_service import HTTPFeedingServiceBase
+
+
+logger = logging.getLogger(__name__)
+
+utcfromtimestamp = datetime.utcfromtimestamp
+
+# HTTP request timeout (seconds)
+DEFAULT_TIMEOUT = 30
+# Maximum number of retries for transient errors (429/5xx)
+MAX_RETRIES = 3
+# Base seconds for exponential backoff (1, 2, 4, ...)
+
+
+BACKOFF_BASE = 1.0
+
+
+class STTContentAPIService(HTTPFeedingServiceBase):
+    """
+    Feeding Service for Superdesk-compatible Content API (/contentapi/items).
+    Simple, Bearer-authorized fetch with incremental since filtering.
+    """
+
+    NAME = "stt_content_api"
+    # Ensure the correct parser is used even if provider.feed_parser is unset
+    FEED_PARSER = "content_api_json"
+    ERRORS = [ParserError.parseMessageError().get_error_description()]
+
+    label = "STT Content API"
+    HTTP_AUTH = False
+
+    fields = [
+        {
+            "id": "url",
+            "type": "text",
+            "label": "Items URL",
+            "placeholder": "https://<host>/contentapi/items",
+            "required": True,
+        },
+        {
+            "id": "api_key",
+            "type": "text",
+            "label": "API Key (Bearer or raw token)",
+            "placeholder": "Bearer <token> OR raw <token>",
+            "required": True,
+        },
+    ]
+
+    def __init__(self):
+        super().__init__()
+        self._session: Optional[requests.Session] = None
+        try:
+            self._session = requests.Session()
+            # Configure automatic retries on the session (built-in urllib3 retries)
+            # urllib3 Retry.total counts *retries* (attempts = retries + 1), so we subtract 1
+            _retry_total = MAX_RETRIES - 1 if MAX_RETRIES > 0 else 0
```

Comment:
not sure I get it, since `MAX_RETRIES` is a constant this will be always the same right? why that `if` ?